adam_config.yaml

```diff
@@ -0,0 +1,51 @@
+axon:
```

Comment:
This shouldn't be added to the main

Reply:
This file is taken out completely now, not needed anymore as it's moved to Adam repo.